.github/workflows/pkg-rust.yml

```diff
@@ -956,10 +962,10 @@ jobs:
         case ${OS_NAME} in
           debian|ubuntu)
             ;;
-          centos|rockylinux)
+          centos|rockylinux|almalinux)
             ;;
           *)
-            echo "::error::This workflow only supports 'debian', 'ubuntu', 'centos' or 'rockylinux' operating systems: '${IMAGE}' is not supported."
```

Comment:
You seem to be adding almalinux as well as rockylinux elsewhere, so why remove rockylinux here?